SpectatorView/LegacyDocs/SharingService/Samples/README.md

```diff
@@ -40,7 +40,8 @@ The apps have a directory "Assets\Addons\HolographicCameraRig" which, alongside
 	![Run In Background](../../../DocumentationImages/Unity/run_in_bg.png)
 
 ## Adding spectator view support to your app
-This sample project has been set up as a shared experience, but if you would like to add this functionality to your existing app, or would like to start from a new project, follow these instructions:
+Since you are using the HoloToolkit sharing service, you must use the old Spectator View Manager code which can be found ![here](https://github.com/Microsoft/MixedRealityCompanionKit/tree/5c9e832304478d408e4eb7712c4eecd7c6460c71/SpectatorView/Samples/SharedHolograms/Assets). This sample project has been set up as a shared experience, but if you would like to add this functionality to your existing app, or would like to start from a new project, follow these instructions:

```

Comment:
Can use the release tag:
https://github.com/Microsoft/MixedRealityCompanionKit/tree/1.5.7.0/SpectatorView/Samples/SharedHolograms/Assets

Reply:
link should not use '!' since it's not an image:
can be found `[here](https://github.com/Microsoft/MixedRealityCompanionKit/tree/1.5.7.0/SpectatorView/Samples/SharedHolograms/Assets)`